src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -107,7 +107,7 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
               />
             ) : (
               <img
-                src={Logo}
+                src={AboutImg}
```

Comment:
You can use a placeholder here instead of using the static image.

Just use the url ```https://via.placeholder.com/200x100``` as image source.

Reply:
This is previous url used now we are using question mark img which free to use.

Reply:
@Sumitmaithani 
```suggestion
                src="https://via.placeholder.com/200x100"
```
That's what I was saying. It would look more modular instead of a question mark image.

Reply:
@palisadoes [here](https://pixabay.com/illustrations/volunteer-hands-tree-grow-2729696/) Free to use under the Pixabay license No attribution required

Reply:
I prefer the pixabay tree because it has a transparent background and it claims to be royalty free. We may have a dark mode in future. Please use that one as a placeholder

Reply:
@palisadoes okay👍